react/InfosCarrousel/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useCallback } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import SwipeableViews from 'react-swipeable-views'
+
+import styles from './styles.styl'
+import Icon from '../Icon'
+
+const InfosCarrousel = ({ children, theme, className, swipleableProps }) => {
+  const count = React.Children.count(children)
+  const [index, setIndex] = useState(0)
+  const nextInfos = useCallback(() => setIndex(index + 1), [index])
+  const previousInfos = useCallback(() => setIndex(index - 1), [index])
```

Comment:
TIL useCallback

Reply:
IMHO, nextInfos and previousInfos, since they are functions should be named with verbs : goToPrevious && goToNext.